deps/ScalableVectorSearch

```diff
@@ -1 +1 @@
-Subproject commit 5dc36cabf5ebd9b76d5b1e17f2349fadfc1e0452
```

Comment:
is this the latest commit? 
@rfsaliev

Reply:
You are right: the submodule should not be changed in this PR as far as VecSim main branch has the latest SVS commit.